lib/rules/media-query-no-invalid/index.mjs

```diff
@@ -143,6 +159,31 @@ const rule = (primary) => {
 	};
 };
 
+/**
+ * @param {import('@csstools/media-query-list-parser').MediaQuery|import('@csstools/media-query-list-parser').GeneralEnclosed} node
+ * @param {Parameters<Rule>[1]} secondaryOptions
+ * @returns {boolean}
+ */
+function shouldIgnoreNode(node, secondaryOptions) {
+	let ignored = false;
+
+	if (secondaryOptions?.ignoreFunctions) {
```

Comment:
[suggestion] Minor. Also here, early return can reduce nesting and make the code a bit readable to me:

```js
	if (!secondaryOptions?.ignoreFunctions) return false;

	let ignored = false;

	// walk...
	
	return ignored;
```